.github/workflows/nightly.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     - uses: gleam-lang/setup-erlang@master
       with:
-        otp-version: 19.3.6.13
+        otp-version: 20.3.8.26
```

Comment:
We should move this one to 21 since there won't be binary compatibility between 20-21, but there will be between 21-24 and it will give us the broader coverage. OTP-20 users can build from source.. Ideally it would make sense to have that on the README as well.

Reply:
b88a958